Sources/Public/CalendarView.swift

```diff
@@ -57,9 +57,12 @@ public final class CalendarView: UIView {
       backgroundColor = .white
     }
 
-    installDoubleLayoutPassSizingLabel()
+    // Must be the first subview so that `UINavigationController` can monitor its scroll position
+    // and make navigation bars opaque on scroll.
     addSubview(scrollView)
```

Comment:
```suggestion
    insertSubview(scrollView, at: 0)
```
makes this code a bit less order dependent and explicit?

Reply:
Oh yes, great suggestion!